Formula/aws-sam-cli-nightly.rb

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+require_relative '../ConfigProvider/config_provider'
+
+class AwsSamCliNightly < Formula
+  include Language::Python::Virtualenv
+
+  config_provider = ConfigProvider.new('aws-sam-cli-nightly')
+
+  desc "AWS SAM CLI 🐿 is a tool for local development and testing of Serverless applications"
```

Comment:
nit: Do we need to mention nightly in the description?

Reply:
Updated.